api/awsauth/awsauth/auth_app.py

```diff
@@ -10,9 +10,6 @@
 import pathlib
 import logging
 
-import sentry_sdk
```

Comment:
Yank out all the Sentry logging because a) we don't track the alerts anymore, and b) removing Sentry as a dependency helps get us below the max bundle size for CloudFront Lambda deploys